### PR TITLE
Add decoder API changes to changelog

### PR DESCRIPTION
**Pull request by lvandeve**
Add these changes to the changelog for version 0.5.x

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,6 +28,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   (`JxlDecoderDCOutBufferSize` and `JxlDecoderSetDCOutBuffer`) and will be
   removed in the next release.
 
+## [0.5] - 2021-07-??
```
Comment (deymo) [inline CHANGELOG.md:31]: This goes in the Unreleased section above

File: CHANGELOG.md [hunk 2]
```diff
@@ -28,6 +28,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+### Changed
+ - Add JxlDecoderRewind and JxlDecoderSkipFrames to decoder API to skip more
```
Comment (deymo) [inline CHANGELOG.md:33]: This looks like it should go in the "Add" subsection (this is new stuff).

File: CHANGELOG.md [hunk 3]
```diff
@@ -28,6 +28,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+   efficiently to earlier animation frames.
+ - Add JxlDecoderSetPreferredColorProfile to decoder API to choose color profile
+   in certain circumstances.
+ - Deprecate JxlDecoderDCOutBufferSize, JxlDecoderSetDCOutBuffer,
```
Comment (deymo) [inline CHANGELOG.md:37]: This goes in the Removed section and I think it is already there.

File: CHANGELOG.md (hunk 1 above)
Comment (lvandeve) [reply CHANGELOG.md]: Done (so, removed mention of 0.5, let me know if the intention is to change the word "Unreleased" into "0.5")

File: CHANGELOG.md (hunk 2 above)
Comment (lvandeve) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 3 above)
Comment (lvandeve) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md (hunk 1 above)
Comment (deymo) [reply CHANGELOG.md]: We haven't release 0.5 yet, only branched v0.5.x. This is a retroactive documentation change only.
Once we release 0.5 (like, right before we do so) we will know the release date and update the header.

File: CHANGELOG.md [hunk 4]
```diff
@@ -10,6 +10,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
  - API: New function to decode the image using a callback outputting a part of a
    row per call.
  - API: 16-bit float output support.
+ - API: JxlDecoderRewind and JxlDecoderSkipFrames functions to skip more
```
Comment (deymo) [inline CHANGELOG.md:13]: nit: we used backticks (\`) around these code function names.

**Review by deymo — APPROVED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (lvandeve) [reply CHANGELOG.md]: Done

**Comment by lvandeve** [comment]
PTAL

**Review by deymo — APPROVED** [review]

**Pull request merged**